storm/src/main/storm/mesos/logviewer/LogViewerController.java

```diff
@@ -96,7 +102,7 @@ protected ProcessBuilder createProcessBuilder() {
         Paths.get(System.getProperty("user.dir"), "/bin/storm").toString(),
         "logviewer",
         "-c",
-        "storm.log.dir=" + System.getenv("MESOS_SANDBOX") + "/logs",
+        "storm.log.dir=" + logdir,
```

Comment:
@erikdw MESOS_SANDBOX is set by Mesos correct? Wondering if we should override `System.getenv("MESOS_SANDBOX")` with logdir only if `storm.log.dir` is configured?